src/hotspot/share/classfile/resolutionErrors.hpp

```diff
@@ -59,7 +59,6 @@ class ResolutionErrorTable : AllStatic {
   // This function is used to encode an invokedynamic index to differentiate it from a
   // constant pool index.  It assumes it is being called with a index that is less than 0
```

Comment:
Is this comment still correct?

Reply:
The last sentence is no longer valid, thanks for catching this!